frontend/utilities/_print.scss

```diff
@@ -48,6 +54,7 @@
     // Remove styling from MOJ sortable table colum headers
     [aria-sort] button {
       display: inline;
+      display: contents;
```

Comment:
This isn't well supported, but does seem to fix the issue of missing table headers in print preview. Although the main issue here is that buttons are used as table headings by the MOJ sortable table component.

Reply:
Yeah, it's a fallback incase `contents` isn't supported at all – which it isn't in some older browsers like IE.

I'll add a comment so that's clear.

Reply:
Added comment.